padertorch/ops/sequence/mask.py

```diff
@@ -60,10 +60,13 @@ def compute_mask(x, sequence_lengths, batch_axis=0, sequence_axis=1):
         batch_axis = x.dim() + batch_axis
     if sequence_axis < 0:
         sequence_axis = x.dim() + sequence_axis
-    sequence_lengths = torch.Tensor(sequence_lengths).long().to(x.device)
+    if not torch.is_tensor(sequence_lengths):
+        sequence_lengths = torch.Tensor(sequence_lengths).long().to(x.device)
+    elif sequence_lengths.device != x.device:
```

Comment:
I think we do not need this elif branch because this should only happen if there is a prior error in the code which might be covered up by this. You can add an assert, that the tensors are on the same device, but I do not feel it is necessary.